.github/workflows/pullrequest.yml

```diff
@@ -0,0 +1,32 @@
+name: Python Lint and Test
+
+on: [pull_request]
```

Comment:
heh, I think there's a misunderstanding. I said both push and pull_request :D

Reply:
Done @hot9cups